src/index.ts

```diff
@@ -165,12 +165,19 @@ export interface Asset {
 
 // Fetch the metadata for the latest stable clangd release.
 export async function latestRelease(): Promise<Release> {
-  const response = await fetch(githubReleaseURL);
-  if (!response.ok) {
-    console.log(response.url, response.status, response.statusText);
-    throw new Error(`Can't fetch release: ${response.statusText}`);
+  const timeoutController = new AbortController();
```

Comment:
per the doc, AbortController was added in node v14.17.0 globally.

It requires a >=v14.17.0 node. I think we should be fine -- clangd vscode extension supports vscode v1.65.0 or later, the vscode v1.65 is based on with the electron v13.5.2 which comes with the node v14.17.0, perfect hit!

Reply:
Thanks for checking! I figured we should avoid introducing a use of a deprecated API if we can avoid it.